src/sstruct_mv/HYPRE_sstruct_mv.h

```diff
@@ -511,6 +511,10 @@ HYPRE_SStructMatrixDestroy(HYPRE_SStructMatrix matrix);
 HYPRE_Int
 HYPRE_SStructMatrixInitialize(HYPRE_SStructMatrix matrix);
 
+HYPRE_Int
+HYPRE_SStructMatrixSetEarlyAssembly( HYPRE_SStructMatrix matrix,
```

Comment:
Here you use the name "Assembly" but in IJ you use "Assemble".

Reply:
Ah, they need to be consistent. Maybe both use "Assemble"?

Reply:
Either is fine with me, but maybe "Assemble" is better because it connects more explicitly to the assemble function.